### ISSUE TITLE
Angular compiler-cli issue

### ISSUE THREAD

**Issue by christiandreher** [urish/angular2-moment#68]
**Bug description**

I localized my app using the `i18n` attribute in a html tag. When I try to execute `ng-xi18n`, I get the following error:

```bash
Error: Unexpected value 'MomentModule' imported by the module 'AppModule'
    at /home/chris/workspace/my-webclient/node_modules/@angular/compiler/bundles/compiler.umd.js:14118:37
    at Array.forEach (native)
    at CompileMetadataResolver.getNgModuleMetadata (/home/chris/workspace/my-webclient/node_modules/@angular/compiler/bundles/compiler.umd.js:14103:46)
    at /home/chris/workspace/my-webclient/node_modules/@angular/compiler/bundles/compiler.umd.js:12947:58
    at Array.forEach (native)
    at OfflineCompiler.analyzeModules (/home/chris/workspace/my-webclient/node_modules/@angular/compiler/bundles/compiler.umd.js:12946:21)
    at Extractor.extract (/home/chris/workspace/my-webclient/node_modules/@angular/compiler-cli/src/extract_i18n.js:92:47)
    at extract (/home/chris/workspace/my-webclient/node_modules/@angular/compiler-cli/src/extract_i18n.js:16:35)
    at Object.main (/home/chris/workspace/my-webclient/node_modules/@angular/compiler-cli/node_modules/@angular/tsc-wrapped/src/main.js:30:16)
    at Object.<anonymous> (/home/chris/workspace/my-webclient/node_modules/@angular/compiler-cli/src/extract_i18n.js:161:9)
Extraction failed
```

This seems to be strongly related: https://github.com/chymz/ng2-ckeditor/issues/36
Obviously `compiler-cli` expects a `.metadata.json` or something. Would be great if you could take a look at this.

**To reproduce:**

1. Create a new angular project
2. Import the `MomentModule` in the main module
3. `npm install --saveDev @angular/compiler-cli`
4. Add a script in `package.json`:  `"extract": "ng-xi18n"`
5. Open `app.component.html` and add the `i18n` attribute somewhere on some tag. E.g: `<h1>App works!</h1>` to `<h1 i18n>App works!</h1>`
6. On project root, run `npm run extract`
7. You should now get the error

**Comment by n-sviridenko** [comment]
+1

**Comment by christiandreher** [comment]
By the way this also prevents one from using any of angulars AoT features

**Comment by Gp2mv3** [comment]
I have the same issue with ioni2 (using AoT).

**Comment by urish** [comment]
Released a fix in 1.0.0-beta.2

**Comment by Gp2mv3** [comment]
Thanks a lot !
